### PR TITLE
Bump envoy to 1.29.3-patch1

### PR DESCRIPTION
**Pull request by davidjumani**
# Description
Bump to the envoy-gloo which builds on the latest envoy v1.29.3

# Context
https://github.com/solo-io/solo-projects/issues/6008

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: changelog/v1.17.0-beta21/bump-envoy.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+changelog:
+- type: DEPENDENCY_BUMP
+  dependencyOwner: solo-io
+  dependencyRepo: envoy-gloo
+  dependencyTag: v1.29.3
+  issueLink: https://github.com/solo-io/solo-projects/issues/6008
+  resolvesIssue: false
+  description: >-
+    Bump envoy-gloo to bring us onto 1.29.3
```
Comment (jenshu) [inline changelog/v1.17.0-beta21/bump-envoy.yaml:9]:
```suggestion
  dependencyTag: v1.29.3-patch1
  issueLink: https://github.com/solo-io/solo-projects/issues/6008
  resolvesIssue: false
  description: >-
    Bump envoy-gloo to bring us onto 1.29.3-patch1
```

**Review by ben-taussig-solo — DISMISSED** [review]

**Review by nfuden — DISMISSED** [review]

**Review by jbohanon — CHANGES_REQUESTED** [review]
Can we revert https://github.com/solo-io/gloo/pull/9307 with this?

**Review by nfuden — APPROVED** [review]
nice revert

**Pull request merged**